src/Microsoft.Health.Dicom.Core/Features/Common/GuidFactory.cs

```diff
@@ -5,7 +5,7 @@
 
 using System;
 
-namespace Microsoft.Health.Dicom.Functions.Client.DurableTask;
+namespace Microsoft.Health.Dicom.Core.Features.Common;
```

Comment:
I am curious, why do we need a factory to get the functionality of Guid.NewGuid()?

Reply:
Mostly so I can inject a deterministic `Guid` value during testing, like `Clock` for `DateTime.Now`

Reply:
Will leave a comment